habitat/plan.sh

```diff
@@ -0,0 +1,275 @@
+# This file is the heart of your application's habitat.
+# See full docs at https://www.habitat.sh/docs/reference/plan-syntax/
+
+# Required.
+# Sets the name of the package. This will be used in along with `pkg_origin`,
+# and `pkg_version` to define the fully-qualified package name, which determines
+# where the package is installed to on disk, how it is referred to in package
+# metadata, and so on.
+pkg_name=goldfish
+
+# Required unless overridden by the `HAB_ORIGIN` environment variable.
+# The origin is used to denote a particular upstream of a package.
+pkg_origin=johnjelinek
+
+# Required.
+# Sets the version of the package.
+pkg_version="0.3.0"
+
+# Optional.
+# The name and email address of the package maintainer.
+pkg_maintainer="Tony Cai"
```

Comment:
@Caiyeon: I put you as the maintainer if you choose to adopt this. That way your CI can build and export docker containers under your `origin`.